.github/workflows/macos.yml

```diff
@@ -36,12 +36,15 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Setup MySQL
+      env:
+        MYSQL_VERSION: ${{ matrix.mysql }}
       run: |
         brew install mysql@${{ matrix.mysql }}
         (unset CI; brew postinstall mysql@${{ matrix.mysql }})
         brew services start mysql@${{ matrix.mysql }}
         sleep 5
         $(brew --prefix mysql@${{ matrix.mysql }})/bin/mysql -uroot -e 'CREATE DATABASE test'
+        [[ "$MYSQL_VERSION" == "8.0" ]] && $(brew --prefix mysql@${{ matrix.mysql }})/bin/mysql -uroot < docker-entrypoint-initdb.d/caching_sha2_password_user.sql
```

Comment:
Should this be any `8.x` version and later? Since we also have versions up to 8.3.0 today for MySQL and each quarter there's a new one.

Reply:
This is the image name and we don't have a matrix for `5.7`, `8.0`, and `8.3`. If we added 8.3 and then this wasn't updated, the build would fail. I'm not against doing a `>=` but it's also not broken currently. 🤷🏼‍♀️